Plugins/Bitbucket/BitbucketPlugin.cs

```diff
@@ -20,7 +23,14 @@ public BitbucketPlugin()
 
         public override bool Execute(GitUIEventArgs args)
         {
-            using (var frm = new BitbucketPullRequestForm(this, Settings, args))
+            Settings settings = Bitbucket.Settings.Parse(args.GitModule, Settings, this);
+            if (settings == null)
+            {
+                MessageBox.Show(_yourRepositoryIsNotInBitbucket.Text);
```

Comment:
Please pass `owner`. Also may worth setting the icon to the error.

Reply:
Done and done.